### DISCUSSION TITLE
Bad Request Error

### DISCUSSION THREAD

**Discussion by EmmiePinckney** [Budibase/budibase#6707] [Help]
Hi friends,

I am a new Budibase user, and still getting used to troubleshooting. I built an employee directory app (using the template of the same name), and although I found the setup fairly simple and fun, I'm experiencing a few odd errors. 

1. **Every time I try to edit my data, budibase crashes.** I uploaded a csv to budibase's internal database, and every time I try to edit, it immediately stops loading anything and everything. I have to close the tab and log in again. (I am self-hosted, if that helps at all.)
2. **Bad Request error.** My screens seem to be working, except that my "save" button on the screen which creates a new row AND on the screen which edits rows returns the "bad request" error. It worked exactly one (1) time, when I was able to change one datapoint, and ever since that first time, it has returned the same error, regardless of what role the user is or who is using the app. 

What can I do to fix this? 

Thanks so much!

**Comment by melohagan** [comment]
Hey @EmmiePinckney 

Thanks for trying out Budibase!

1. There's possibly a formatting issue with the CSV that Budibase doesn't like. 
    Do you see any errors in the console/network tabs?

2. Maybe there's a unique constraint violation. I would need to see the full bad request error which will be in the network tab response. Fn+F12 on Chrome.
    I've raised an issue here for more detailed error messages: https://github.com/Budibase/budibase/issues/6356
